content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+Context is defined as the verses or verses that are required.
+
+##### Finish-the-Verse
+Begins at the beginning of a single verse and finishes at the end of that verse.
+The Question Type abbreviation is "FTV"
+
+##### Finish-This
+Begins in the middle of a verse and finishes at the end of the verse.  A FT must start at the beginning of a sentence or the beginning of a quotation.
+The Question Type abbreviation is "FT"
+
+##### Finish-these-Two-Verses
+Begins at the beginning of a verse and continues to the end of the next verse. Used when a verse is not strong enough on its own and needs another verse to explain it or can be used when two verses combined form an even stronger thought. 
+The Question Type abbreviation is "F2V"
+
+##### Finish-This-and-the-Next
+Begins in the middle of a verse and finishes at the end of the next verse.  A FTN must start at the beginning of a sentence or the beginning of a quotation. Used when part of a verse is not strong enough on its own and needs another verse (or part of a verse) to explain it or can be used when two verses combined form an even stronger thought.
+The Question Type abbreviation is "FTN"
+
+FTV questions should not start out with an insignificant introduction such as “He replied” or “He said to them”.  These verses, if spiritually significant, should be used as FT or FTN type questions.  For example, Luke 11:28 says, He replied, “Blessed rather are those who hear the word of God and obey it.”  This verse could be used as an excellent FT question starting with “Blessed rather are those who …”. 
+
+#### Situation
+Used only in the year where the Material Type is Narrative
+Contains two parts to the question:
+- Quotation
+    - Must be a direct quotation from Scripture by a person or an entity. 
+    - The quotation must be reasonable in length, not exceeding two consecutive verses. 
+    - The first five words must be unique to all Quotations.
+    - The quizzer is not required to repeat the quotation word-perfect but must be substantially close without leaving out key components of the quotation.
+    - Must start at the beginning of a quotation or the beginning of a sentence within the quotation.  When a Situation question comes from a quotation that runs for more than 2 verses, the beginning of the Situation question must start within the first 2 verses of the quotation.
+    - If the Quotation is split-up by non-quotation words, the Quotation will be read without the non-quotation words.
+        - A situation question can combine two separate quotations as long as no words are added and the two quotes flow easily from one to another.  For example, Matthew 4:19 says, “Come, follow me”, Jesus said, “and I will send you out to fish for people.”  This example could be written, Who said it?  “Come, follow me, and I will send you out to fish for people” Answer:  Jesus.
+- Questions regarding the Situation of the Quotation
```

Comment:
Thoughts on wording?? The whole type is called a "Situation Question." So I didn't know what to call the "interrogative questions about the quotation"

Reply:
I think this is fine.

Reply:
Agreed.